### DISCUSSION TITLE
What is the default font in jsPsych?

### DISCUSSION THREAD

**Discussion by NickWasted** [jspsych/jsPsych#3087] [Q&A] [answered]
What` is the default font in jsPsych? It might seem a dumb question, but I couldn't find any info about it.

For example, if my stimulus is:
`"<p style='color:red;font-size:32pt;'>Hello</p>"`

what is its font? Arial?

**Comment by jodeleeuw** [comment, marked as answer]
If you are using the jspsych.css stylesheet then the default font is Open Sans.

https://github.com/jspsych/jsPsych/blob/8f0075b8a17bde966f2d0603b236d1cef076fa57/packages/jspsych/src/index.scss#L46-L59

**Reply by jodeleeuw** [reply]
Also, we automatically bundle the Open Sans font with the stylesheet, so it should appear as Open Sans for everyone, regardless of whether they have the font on their system.

**Reply by NickWasted** [reply]
Nice, thanks!
